lib/_emerge/AbstractEbuildProcess.py

```diff
@@ -390,9 +391,12 @@ def _elog(self, elog_funcname, lines):
         elog_func = getattr(elog_messages, elog_funcname)
         global_havecolor = portage.output.havecolor
         try:
-            portage.output.havecolor = self.settings.get(
-                "NOCOLOR", "false"
-            ).lower() in ("no", "false")
+            if not self.settings.get("NOCOLOR"):
+                portage.output.havecolor = self.settings.get("NO_COLOR") == ""
```

Comment:
What happens if `NO_COLOR` is unset?

Reply:
I need to change it here too sorry but havecolor would be true. I had this in the back of my mind too ( then I convinced myself that it might be an expected outcome ) Should I change that ?